web/README.md

```diff
@@ -0,0 +1,124 @@
+| Operation | Description | Example Scripts | 
+| --------- | --------- | ----------------------- | 
+| `REVIEW`  | Review coop proposal. | [test_coopproposal_review.sh](./scripts/test_coopproposal_review.sh)  | 
+
+### Data Flow
+| Operation | Step 1: Create Proposal | Step 2: Proposal Reviewed. Rejected. | Step 2: Proposal Reviewed. Approved. |
+| --------- | ----------------------- | ------------------------------------ | ------------------------------------ |
+| Create New Coop | ![](./img/CreateOperation_Proposal.png) | ![](./img/CreateOperation_ReviewRejected.png) | ![](./img/CreateOperation_ReviewApproved.png) |
+| Update Existing Coop | ![](./img/UpdateOperation_Proposal.png) | ![](./img/UpdateOperation_ReviewRejected.png) | ![](./img/UpdateOperation_ReviewApproved.png) |
+| Delete Existing Coop | ![](./img/DeleteOperation_Proposal.png) | ![](./img/DeleteOperation_ReviewRejected.png) | ![](./img/DeleteOperation_ReviewApproved.png) |
+
+## Testing
+Tests are stored in the [directory/tests](./directory/tests/) directory using Python's `unittest` framework with additional add-ins from Django and Django Rest Framework. Tests can be run via Django's [manage.py](./manage.py) script. 
+
+Run all tests:
+```
+python manage.py test directory.tests
+```
+
+Run all tests in a specific file:
+```
+python manage.py test directory.tests.[file_name]
+
+Example:
+python manage.py test directory.tests.test_coop_list
+```
+
+Run all tests in a specific class:
+```
+python manage.py test directory.tests.[file_name].[class_name]
+
+Example:
+python manage.py test directory.tests.test_coop_list.TestCoopList
+```
+
+Run a single test:
+```
+python manage.py test directory.tests.[file_name].[class_name].[test_name]
+
+Example:
+python manage.py test directory.tests.test_coop_list.TestCoopList.test_list_all
+```
+
+## TODO List
+- [x] *Simplify CoopAddressTag Serializer Object Structure
+- [x] CoopAddressTag update (serializer)
+- [x] Add more Coop -> CoopAddressTag test cases
+- [x] Search Coops (views, models)
+- [x] Simplify Address Data Structure.
+- [x] *Routes for other objects
+- [x] Turn on new authentication method (JWT Tokens)
+- [x] Create User (views)
+- [x] Create User Test cases
+- [x] Change Password Workflow
+- [x] Enabled filter in coop list (views)
+- [x] LocationService: Add coordinates to Address (serializer)
+- [x] LocationService: Ratelimit and retry for api.
+- [x] Add county to model. Look up and apply from geocode. 
+- [x] Coop.patch Proposed Changes (views)
+- [x] What should be shown on is_public vs not?
+- [x] What to do about spreadsheet?
+- [x] Move API to /api/v1/ endpoint.
+- [X] Docker Install: PROD vs DEV
+- [X] ENV file for PRD
+- [x] SMTP Settings: PROD vs DEV
+- [x] Confirm `coops/<pk>` is really working off of public_id.
+- [x] Confirm `coops/public` is not needed.
+- [x] Create endpoint for All proposals for given coop.
+- [x] Add data endpoint back in. 
+- [x] What is monkey_patching.py?
```

Comment:
I'd also like to know what `monkey_patching.py` is. Did you figure it out?

Reply:
It extended the country and state models so that you could look up entities by their name. Kind of redundant since Django's queryset engine can do that for you. But not needed once I moved those lookups out of the database into statically defined views.